src/main/java/seedu/address/logic/parser/RecipeBookParser.java

```diff
@@ -55,6 +67,12 @@ public Command parseCommand(String userInput) throws ParseException {
             return new ListCommand();
         case ViewCommand.COMMAND_WORD:
             return new ViewCommandParser().parse(arguments);
+        case ClearCommand.COMMAND_WORD:
+            System.out.println(arguments);
```

Comment:
I think you forgot to remove this line when testing/implementing 😆 

P.S should try and use debugger hehehehehehehehe

```suggestion
```